constructor/osx/clean_cache.sh

```diff
@@ -4,7 +4,7 @@
 
 # $2 is the install location, which is ~ by default
 # but which the user can change.
-set -x
+set -eux
```

Comment:
Same here, I'd rather use the same flags (with or without `x` depending on the use case, but same error signalling flags).

Reply:
see above sh vs bash set options... x was given, only eu get added due to sh.